machine_learning_hep/analysis/analyzer_jet.py

```diff
@@ -15,33 +15,24 @@
 """
 main script for doing final stage analysis
 """
-# pylint: disable=too-many-lines
+# pylint: disable=too-many-lines, bad-whitespace, line-too-long
```

Comment:
Would be better to find the "bad-whitespace" and remove this "bad-whitespace". That is much better. Because this will ignore any white space style issues in the future

Reply:
Will be fixed later.